### ISSUE TITLE
Error 307

### ISSUE THREAD

**Issue by Zamundaaa** [ValveSoftware/SteamVR-for-Linux#195]
#### Your system information
* Steam client version (build number or date): 1555457005
* Distribution (e.g. Ubuntu): Manjaro KDE
* Graphics driver version (run nvidia-settings): rx580 with mesa 19.0.3
* Gist for SteamVR System Information: https://gist.github.com/Zamundaaa/48250d9eab5acc776cdf3f9480d7299f
* Opted into Steam client beta?: [Yes/No] yes
* Opted into SteamVR beta?: [Yes/No] yes
* Have you checked for system updates?: [Yes/No] yes

Basically every time I close an app SteamVR crashes with the compositor connect failed error 307 message popup. Over the SteamVR status window it shows something about fatal error, error code -202. SteamVR home doesn't seem to be affected (when launching another app at least)

**Comment by Zamundaaa** [comment]
No reponse yet?
This makes my Vive setup pretty much unusable. It's not just in the beta.

**Comment by h1z1** [comment]
Likely more there's a lot to go through in those logs.

``No GPU timing Vulkan device in CVRCompositorSharedTextures::BeginGPUTimingCommandBuffer!``

Rings a bell but I can't remember why.  Have you tried a fresh install of steam?

**Comment by Zamundaaa** [comment]
I've tried a reinstall of SteamVR, but that didn't help. I'm gonna try a complete reinstall of Steam now if you think it may help.

**Comment by Zamundaaa** [comment]
Whelp. After reinstalling Steam & installing SteamVR it immediately crashed with error -301.
Trying the beta worked though, no more crashes. Thanks for your help! (This was easier to fix than I thought...)

**Comment by Zamundaaa** [comment]
Okay... too soon. After steam had finished downloading the workshop content for SteamVR it began to crash as before. Or at least I think that's the cause...
I'll begin removing stuff tomorrow, hopefully it is just this...

**Comment by Zamundaaa** [comment]
I've now removed all the workshop content and it seems to somewhat work now.
It doesn't crash every time now, and when it does it seemingly can recover somtimes.
When the crash occurs it says "starting SteamVR Home" and an error message -203 and then takes a while, SteamVR actually gets started and the error message disappears, everything works fine.
The last time I tried though it actually crashed completely, this time with an actual error popup shared IPC compositor init failed (303).
This is getting very strange. There shouldn't be any init happening, should it? SteamVR is already running...
Here's a system report from after the fail:
https://gist.github.com/Zamundaaa/3b6883d503f4f9cf36473e2c9a396bc0
